lib/rubocop/rspec_rails/shared_contexts/default_rspec_language_config_context.rb

```diff
@@ -1,15 +1,15 @@
 # frozen_string_literal: true
 
-RSpec.shared_context 'with default RSpec/Language config' do
+RSpec.shared_context 'with default RSpecRails/Language config' do
```

Comment:
I think we had a ticket to expose this in lib in rubocop-rspec. Can we reuse that shared example from there and remove it completely here?